Source/ACE.Common/ThreadSafeRandom.cs

```diff
@@ -38,5 +40,18 @@ public static uint Next(uint min, uint max)
                 return (uint)random.Next((int)min, (int)(max + 1));
             }
         }
+        public const string CookieChars = "0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ";
```

Comment:
This prob doesn't need to go into ThreadSafeRandom and can stay local to the TransferServer namespace